packages/core/src/components/TextField/__stories__/TextField.mdx

```diff
@@ -123,8 +123,6 @@ Users can insert text.
 
 ### Required input field
 
-Use the <code>requiredAsterisk</code> prop to indicate that a field is required. When set to <code>true</code>, an asterisk (\*) will appear next to the label.
```

Comment:
shouldn't be changed to `required`? and not to be removed entirely?

Reply:
`required` already exists

Reply:
I meant in this story

Reply:
Oh, I think it's self explanatory so I removed it entirely, WDYT?

Reply:
sounds good, it also contains the code with required so sound cool